docs/madr/decisions/008-mesh-logging.md

```diff
@@ -0,0 +1,381 @@
+  from:
+    - targetRef:
+        kind: MeshService
+        name: web-frontend
+```
+
+This will log all the inbound traffic from `web-backend` in `web-backend`.
+
+#### Only outbound
+
+```yaml
+type: MeshLogging
+mesh: default
+spec:
+  targetRef:
+    kind: MeshService
+    name: web-backend
+  to:
+    - targetRef:
+        kind: MeshService
+        name: web-queue
+```
+
+This will log all the outbound traffic to `web-queue` in `web-backend`.
+
+#### Both inbound and outbound
+
+```yaml
+type: MeshLogging
+mesh: default
+spec:
+  targetRef:
+    kind: MeshService
+    name: web-backend
+  from:
+    - targetRef:
+        kind: MeshService
+        name: web-frontend
+  to:
+    - targetRef:
+        kind: MeshService
+        name: web-queue
+```
+
+This will log all the inbound traffic from `web-frontend` and outbound traffic to `web-queue` in `web-backend`.
+
+### Backends
+
+#### Keep backends in the `MeshLogging` resource for now, move them into `MeshLoggingBackend` policy in the future
+
+For now new Policy `MeshLogging` will reuse the backend definitions from `Mesh`.
+
+```yaml
+apiVersion: kuma.io/v1alpha1
+kind: Mesh
+metadata:
+  name: default
+spec:
+  logging:
+    # MeshLogging policies may leave the `backend` field undefined.
+    # In that case the logs will be forwarded into the `defaultBackend` of that Mesh.
+    defaultBackend: file
+    # List of logging backends that can be referred to by name
+    # from MeshLogging policies of that Mesh.
+    backends:
+      - name: logstash
+        type: tcp
+        conf:
+          address: 127.0.0.1:5000
+      - name: file
+        type: file
+        conf:
+          path: /tmp/access.log
+---
+type: MeshLogging
+mesh: default
+spec:
+  targetRef:
+    kind: MeshService
+    name: web-backend
+  from:
+    - targetRef:
+        kind: MeshService
+        name: web-frontend
+      default:
+        backend: logstash # Forward the logs into the logging backend named `logstash`.
+```
+
+In the future we will introduce `MeshLoggingBackend` policy that would hold that data.
+Definitions there will have precedence over definitions in `Mesh`.
+In the future the user will re-create (or maybe we could do this automatically) equivalent `MeshLoggingBackend` policies.
+
+##### Positive Consequences 
+
+* easier to implement
+
+##### Negative Consequences
+
+* breaks the barrier between new / old policies by reusing old policies resources
+* possibly more work for the user in the future
+
+#### Move backend to the `MeshLoggingBackend` policy now
+
+Moving backend to a new policy called `MeshLoggingBackend` immediately.
+That policy would be neither `inbound` nor `outbound` it would just store backend definitions.
+It would support `targetRef` of `Mesh` but could probably support more in the future.
+This allows us to have consistent naming (`enabledBackend`) across all policies and
+would make it easier to have per zone backends.
+
+```yaml
+type: MeshLoggingBackend
+mesh: default
```

Comment:
`MeshLoggingBackend` and `GlobalMeshLoggingBackend` seems to much, I'd personally go with global scoped. What others think?

Reply:
from @lobkovilya - we have "backend" concept in 3 other policies so this might be out of scope of this PR. For now the best solution would be to re-use backends from `Mesh` and then rely on the solution provided for the common `backend` approach.